### DISCUSSION TITLE
Feature request: Configurable OLED brightness

### DISCUSSION THREAD

**Discussion by Mr-Blinky** [steadramon/ESPGeiger#20] [Ideas]
Feature request: Configurable OLED brightness - a setting to set the brightness of the OLED.

I think this requires little explanation, but essentially a form field in the 'config' page for setting the OLED brightness.

Any counters that I have in my office, around me, are set to have their displays on 100% of the time. With certain devices this does cause the OLED to burn out prematurely. Having the ability to control the brightness of the OLED would allow me to still have the display on all of the time, but at a lower level so the OLED lasts longer.

As it happens, the brightness that is hard coded (I assume) into ESPGeiger isn't that bad. It's not too bright, but could be better, it could go dimmer. For me, it doesn't need to be as bright as it is currently. I have a NetIO GC10next with the same OLED and that is destroyed after (just under) two years of use, as that is too bright. There is a big trade-off between brightness and longevity of these cheap OLEDS, so running them dimmer will most certainly increase longevity.

For what it is worth; the ESPGeigers that I plan of releasing out in the wild won't have their OLEDs on all the time. They will use the button to enable them as and when. But the counters I have in my office will always remain with their displays on all of the time.

Cheers, Blinky

**Comment by steadramon** [comment]
This should be available in the next release

**Comment by Mr-Blinky** [comment]
I'm closing this as this feature was introduced some time back.
